doc/source/whatsnew/v0.25.0.rst

```diff
@@ -1188,6 +1188,7 @@ Other
 - Allow :class:`Index` and :class:`RangeIndex` to be passed to numpy ``min`` and ``max`` functions (:issue:`26125`)
 - Use actual class name in repr of empty objects of a ``Series`` subclass (:issue:`27001`).
 - Bug in :class:`DataFrame` where passing an object array of timezone-aware `datetime` objects would incorrectly raise ``ValueError`` (:issue:`13287`)
+- Trying to set the display.precision, display.max_rows or display.max_columns using set_option to anything but a None or a positive int will raise a ``ValueError`` (:issue:`23348`)
```

Comment:
can you move this v0.26.0 docs (pushing shortly)

Reply:
I don't see a v.0.26.0.rst file out there yet.  Is there a template I should use to create one?

Reply:
Use the 1.0 file.